web/src/features/moreCast2/components/TabbedDataGrid.tsx

```diff
@@ -236,7 +236,7 @@ const TabbedDataGrid = ({ morecast2Rows, fromTo, setFromTo, fetchWeatherIndeterm
       handleClose: handleClose
     })
     // Provide default with all columns
-    const showHideColumnsUngroupedState = weatherModelColumns.map((column: GridColDef): ColumnVis => {
+    const showHideColumnsUngroupedState = weatherModelColumns?.map((column: GridColDef): ColumnVis => {
```

Comment:
Are these possibly undefined now? Maybe the react types changed?

Reply:
I was trying to debug unit tests which complained of .map being called in undefined and this got left behind as an artifact of my debugging. 

I'll revert the change. Thanks!